CHANGELOG.rst

```diff
@@ -268,6 +268,57 @@ double check with the list of breaking changes in mind.
 * Improved documentation on embedding the SDK
 * [#921] Added decision tree docs
 
+1.1.6 (2022-09-29)
+==================
+
+Bugfix release + preparation for 2.0.0 upgrade
+
+* [#1856] Fixed crash on logic rule saving in the admin
+* [#1842] Fixed crash on various types of empty StUF-BG response
+* [#1832] Prevent and handle location service rate limit errors
+* [#1960] Ensure design tokens override default style
+* [#1957] Fixed not being able to manually retry errored submission registrations having
+  exceeded the retry limit
+* [#1867] Added more StUF-ZDS/ZGW registration fields.
+* Added missing translation for max files
+* [#2011] Worked around thread-safety issue when configuring Ogone merchants in the admin
+* [#2066] Re-added key validation in form builder
+
+**Upgrade preprations**
```

Comment:
No need to point this out here or make a separate heading for it. It's relevant for 2.0.0 where this is mentioned

Reply:
I'll remove it :+1: